### ISSUE TITLE
Type issue inside addStorage 

### ISSUE THREAD

**Issue by dhoko** [crowdin/crowdin-api-client-js#227]
hey :wave: 

I spent quite a few minutes trying to understand why my upload failed. 
I was sending a JSON file and got a ... `type: chrome` from the API instead of the `custom_fomat` I was expecting (_what I have if I upload via the UI_).

Based on the source I had no idea what the lib wanted to have

- cf https://github.com/crowdin/crowdin-api-client-js/blob/master/src/uploadStorage/index.ts#L953

So I looked into the tests and ok, you use a string https://github.com/crowdin/crowdin-api-client-js/blob/master/tests/uploadStorage/api.test.ts#L21

nope :grin: 

We must send a buffer in order to have the API doing its magic to get the correct type value (_as sending a `type=custom_format` === 500 from the API_)

Based on the API doc:
> string (Binary file data) File content. Text of file or any other raw binary data.

I would expect a buffer

-------------------------------------------------

Can you add a custom type and describe what we can use, and what to expect? 

Thx :pray:

**Comment by andrii-bodnar** [comment]
Hi @dhoko,

try using the `fs.readFileSync` following the example in #209 for screenshots upload

You will need only to change the `fileName` and `contentType` depending on your file

**Comment by dhoko** [comment]
Hum I looked at the debugger before changing the string to a buffer and they both have the correct content type, only diff between the 2 requests is the type of data inside the request.

I changed my code from:

```js
  const storage = await uploadStorageApi.addStorage(path.basename(crowdinFile), await readFile(sourceFile, 'utf-8))
``` 

to 
```js
  const storage = await uploadStorageApi.addStorage(path.basename(crowdinFile), await readFile(sourceFile))
```

**Comment by yevheniyJ** [comment]
Fixed in v1.21.2
Now you can pass either `Buffer` or `string` and for both it will produce correct type

```js
const contentString = fs.readFileSync(file, 'utf8');
const contentBuffer = fs.readFileSync(file);
//both options now will produce file with correct type
await client.uploadStorageApi.addStorage(file, contentBuffer);
//or
await client.uploadStorageApi.addStorage(file, contentString);
```
